backend/LexBoxApi/GraphQL/LexMutations.cs

```diff
@@ -129,10 +129,10 @@ public async Task<User> ChangeUserAccountData(ChangeUserAccountDataInput input,
     [UseMutationConvention]
     public async Task<int> ChangeUserAccountByAdmin(ChangeUserAccountByAdminInput input, LexBoxDbContext dbContext)
     {
-        var admin = await dbContext.Users.FindAsync(input.AdminId);
+        var admin = await dbContext.Users.FindAsync(_loggedInContext.User.Id);
```

Comment:
I guess I meant: we don't need to query for the user at all, beacuse we already have it.

Reply:
You mean remove FindAsync?
Don't we need it so we can do ```if (admin.IsAdmin...```

Reply:
Can I leave that out if i have ```[AdminRequired]```?

Reply:
Yeah, totally 👍